### ISSUE TITLE
pipfile just support pipenv?

### ISSUE THREAD

**Issue by 625781186** [pypa/pipfile#119]
![image](https://user-images.githubusercontent.com/30521429/61837322-d443ba00-aeb6-11e9-9d47-c00082729cb9.png)

**Comment by di** [comment]
See https://github.com/pypa/pipfile#pip-integration-eventual

**Comment by 625781186** [comment]
@di
Sorry , I get no useful information about my problem.
If you can, I'd like you to tell me the answer directly.

**Comment by pradyunsg** [comment]
Yes. Right now, only pipenv supports it. pip may support Pipfile in the future at some point.

**Comment by filips123** [comment]
@di Is there any update?

This repository wasn't active for a year now and Pip still doesn't support Pipfile. As this would be very nice addition, I would like to see it supported and official soon.

**Comment by cclauss** [comment]
https://pipenv.pypa.io is now a dead link and PyPA has removed all mention of pipfile... https://www.pypa.io/en/latest/search/?q=pipfile
